packages/loot-core/src/shared/util.js

```diff
@@ -319,10 +315,6 @@ export function amountToCurrency(n) {
   return numberFormat.formatter.format(n);
 }
 
-export function amountToPrettyCurrency(n, code) {
```

Comment:
Unused fn.

Reply:
Yep, I was planning on tearomg this out as well. There's a fair few functions in here that are unused.